src/main/java/org/intellij/sdk/codesync/ProjectOpenCloseListener.java

```diff
@@ -105,6 +106,12 @@ public void projectOpened(@NotNull Project project) {
       // Populate state for all the opened modules. Module is the term used for projects opened using "Attach" option
       // in the IDE open dialog box.
       for (VirtualFile contentRoot: contentRoots) {
+
+        if(Utils.isIndividualFileOpen(contentRoot.getPath())){
+          NotificationManager.notifyError(INDIVIDUAL_FILE_OPENED);
```

Comment:
You can skip the notification here. The reason is that this code runs in the background and does not execute in response to user input.

User might be confused as to why we are saying that repo can not be synced because it is a file since he did not ask to sync the repo.